src/otx/backend/openvino/models/multiclass_classification.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Class definition for classification model entity used in OTX."""
+
+from __future__ import annotations
```

Comment:
same as above, do we still need this import?

Reply:
Yes, it is required by ruff 
![image](https://github.com/user-attachments/assets/269f319e-b7c7-48c6-a28b-803bfb906cdc)